ansible/group_vars/nimbus.kiln.yml

```diff
@@ -38,8 +38,10 @@ geth_websocket_port: '{{ 9546 + idx|int + 1 }}'
 beacon_node_service_name: 'beacon-node-{{ beacon_node_network }}-devauth-{{ "%02d"|format(idx|int+1) }}'
 beacon_node_network: 'kiln'
 beacon_node_proposer_boosting_debug: true
-# FIXME: Pinned because bf127920 breaks the testnet config.
-beacon_node_repo_branch: 'a35c5f83a8f7880165e19784f73463663db9fdd8'
+beacon_node_repo_branch: 'kiln-dev-auth'
+beacon_node_light_client_data_enabled: true
+beacon_node_light_client_data_serve: true
+beacon_node_light_client_data_import: 'only-new'
```

Comment:
Please move the light client variables below the eth1 sync config and separate with a comment for clarity.

Reply:
Done